sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ClaimsBasedSecurityChannelTest.java

```diff
@@ -107,8 +107,7 @@ public void teardown() {
     @Test
     public void authorizesSasToken() {
         // Arrange
-        // Subtracting two minutes because the AccessToken does this internally.
-        final Date expectedDate = Date.from(validUntil.minusMinutes(2).toInstant());
+        final Date expectedDate = Date.from(validUntil.toInstant());
```

Comment:
@jianghaolu  is this something we're overriding ?

Reply:
Yes because AccessToken no longer has 2 minutes built-in.